datahub-classify/tests/expected_output/expected_infotypes_UNIT_TESTING.json

```diff
@@ -7,7 +7,7 @@
         "Person": "Full_Name",
         "Current Age": "Age",
         "Retirement Age": "Age",
-        "Gender": "Gender",
+        "gender": "Gender",
```

Comment:
why did this one become lowercase and none of the other ones did?

Reply:
The column name was changed to "gender" from "Gender" in the public dataset. Hence, it was changed in the JSON files as well.